test/party_foul/issue_renderers/rails_test.rb

```diff
@@ -33,7 +35,24 @@
   end
 
   describe '#session' do
-    let(:params) { {'action_dispatch.parameter_filter' => ['password'], 'rack.session' => { 'status' => 'ok', 'password' => 'test' }, 'QUERY_STRING' => { 'status' => 'fail' } } }
+    def rails_session(hash_data)
+      store = Class.new {
```

Comment:
Credit: adopt this session store fake from rails test codes https://github.com/rails/rails/blob/master/actionpack/test/dispatch/request/session_test.rb#L110-L114